docs/nodes/run/with-installer/advance-config-maintenance.md

```diff
@@ -0,0 +1,212 @@
+options and their usage:
+
+- `admin` - [Admin API](/reference/avalanchego/admin-api.md) will be enabled
+- `archival` - disables database pruning and preserves the complete transaction history
+- `state-sync` - if `on` state-sync for the C-Chain is used, if `off` it will
+  use regular transaction replay to bootstrap; state-sync is much faster, but
+  has no historical data
+- `db-dir` - use to provide the full path to the location where the database
+  will be stored
+- `fuji` - node will connect to Fuji testnet instead of the Mainnet
+- `index` - [Index API](/reference/avalanchego/index-api.md) will be
+  enabled
+- `ip` - use `dynamic`, `static` arguments, of enter a desired IP directly to be
+  used as the public IP node will advertise to the network
+- `rpc` - use `any` or `local` argument to select any or local network interface
+  to be used to listen for RPC calls
+- `version` - install a specific node version, instead of the latest. See
+  [here](#using-a-previous-version) for usage.
+
+Please note that configuring `index` and `archival` options on an existing node
+will require a fresh bootstrap to recreate the database.
+
+Complete script usage can be displayed by entering:
+
+```bash
+./avalanchego-installer.sh --help
+```
+
+### Unattended Installation
+
+If you want to use the script in an automated environment where you cannot enter
+the data at the prompts you must provide at least the `rpc` and `ip` options.
+For example:
+
+```bash
+./avalanchego-installer.sh --ip 1.2.3.4 --rpc local
+```
+
+### Usage Examples
+
+- To run a Fuji node with indexing enabled and autodetected static IP:
+
+    ```bash
+    ./avalanchego-installer.sh --fuji --ip static --index
+    ```
+
+- To run an archival Mainnet node with dynamic IP and database located at `/home/node/db`:
+
+    ```bash
+    ./avalanchego-installer.sh --archival --ip dynamic --db-dir /home/node/db
+    ```
+
+- To use C-Chain state-sync to quickly bootstrap a Mainnet node, with dynamic IP and local RPC only:
+
+    ```bash
+    ./avalanchego-installer.sh --state-sync on --ip dynamic --rpc local
+    ```
+
+- To reinstall the node using node version 1.7.10 and use specific IP and local RPC only:
+
+    ```bash
+    ./avalanchego-installer.sh --reinstall --ip 1.2.3.4 --version v1.7.10 --rpc local
+    ```
+
+## Node Configuration
+
+File that configures node operation is `~/.avalanchego/configs/node.json`. You
+can edit it to add or change configuration options. The documentation of
+configuration options can be found
+[here](/nodes/configure/avalanchego-config-flags.md). Configuration may look like
+this:
+
+```json
+{
+  "public-ip-resolution-service": "opendns",
+  "http-host": ""
+}
+```
+
+Note that configuration file needs to be a properly formatted `JSON` file, so
+switches are formatted differently than for command line, so don't enter options
+like `--public-ip-resolution-service=opendns` but as in the example above.
```

Comment:
```suggestion
Note that the configuration file needs to be a properly formatted `JSON` file, so
switches should formatted differently than they would be for the command line. 
Therefore, don't enter options like `--public-ip-resolution-service=opendns` as shown 
in the example above.
```

corrected the grammar be a bit more clear, but still wondering what this is really trying to say?